docs/cookbook/recipe_custom_action.rst

```diff
@@ -270,3 +270,70 @@ The full ``CarAdmin.php`` example looks like this:
     parameter, is not involved at all, and then you will get an error "There is no
     _sonata_admin defined for the controller
     AppBundle\Controller\XxxxCRUDController and the current route ' '."
+
+Custom Action without Entity
+----------------------------
+
+Creating an action that is not connected to an Entity is also possible.
+Let's imagine we have an import action. We register our route::
+
+    // ...
+    use Sonata\AdminBundle\Route\RouteCollection;
+
+    protected function configureRoutes(RouteCollection $collection)
+    {
+        $collection->add('import');
+    }
+
+We add the controller action::
+
+    use Sonata\AdminBundle\Controller\CRUDController as Controller;
+    use Symfony\Component\HttpFoundation\Request;
+
+    class CRUDController extends Controller
+    {
+        public function importAction(Request $request)
+        {
+            //do your import logic
+        }
+
+Now, instead of adding the action to the form mapper, we can add it next to
+the add button. In your admin class, overwrite ``configureActionButtons``
```

Comment:
"the ``configureActionButtons``"

Reply:
Come onnnnnnn, is this torture? You come every 20h and find one wrong word :smile:

Reply:
:smiling_imp: